.bazelrc

```diff
@@ -1,3 +1,3 @@
-build:Linux --cxxopt='-std=c++17'
-build:macOS --cxxopt='-std=c++17'
-build:Windows --cxxopt='/std:c++17'
+build:linux --cxxopt='-std=c++17'
+build:macos --cxxopt='-std=c++17' --features=-supports_dynamic_linker
```

Comment:
Is there a reason why these flags are added to macOS only?

Reply:
Yes, because they are only needed for macOS